### ISSUE TITLE
[BUG] Unable to create new zones based on template

### ISSUE THREAD

**Issue by andrewhotlab** [WillyXJ/facileManager#508]
Replace everything between stars with current version of your facileManager and module installations:  
fM Version : **4.0.2**
fmDNS Version : **4.0.1**

**In raising this issue, I confirm the following:**

- [x] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x] I have checked that the issue I'm posting isn't already reported.
- [x] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---

I do not know from if this issue arose in the very last release or if it has being present since more time, but I just noticed (and able to reproduce on different FM instances) with the latest one. When I try to create a new forward zone based on a template, the web interface returns this error:

```
Database error: [Column 'domain_dnssec_signed' cannot be null | Query: [INSERT INTO `fm_dns_domains` (domain_template_id,domain_name,domain_mapping,domain_dynamic,domain_dnssec,domain_dnssec_sig_expire,domain_dnssec_generate_ds,domain_dnssec_sign_inline,domain_dnssec_signed,domain_dnssec_parent_domain_id,domain_view,domain_type,domain_name_servers,account_id) VALUES ('5','mydomain.name','forward','no','no','0','no','no',NULL,'0','0','master','g_1','1')]]
```

No problem if you try to add a new zone without connecting it to any template.

**Comment by WillyXJ** [comment]
This is now fixed in [fmDNS 4.0.2](http://www.facilemanager.com/modules/fmdns/) and later.

**Comment by andrewhotlab** [comment]
Wonderful, thank you so much @WillyXJ for your work in general, and for this quick response!
